src/grokcore/component/compat.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import martian
+from zope.interface import classImplements
+from zope.interface.interfaces import IInterface
+
+if sys.version_info[0] < 3:
+    from zope.interface import implements
+else:
+
+    class implements(martian.Directive):
+        scope = martian.CLASS
+        store = martian.ONCE
+        default = None
+        
+        def factory(self, *interfaces):
+            for interface in interfaces:
+                if not IInterface.providedBy(interface):
+                    raise Exception('%s is not a interface' % repr(interface))
+            return interfaces
+
+
+    class ClassImplementerGrokker(martian.ClassGrokker):
```

Comment:
There were a lot more versions that needed to be pinned and added to the buildout.cfg:
[versions]
six = 1.7.3
zope.component = 4.2.1
zope.configuration = 4.0.3
zope.event = 4.0.3
zope.exceptions = 4.0.7
zope.i18nmessageid = 4.0.3
zope.interface = 4.1.1
zope.schema = 4.4.1
zope.testing = 4.1.3
zope.testrunner = 4.4.3
martian = 0.15
zc.buildout = 2.2.1
zc.recipe.egg = 2.0.1
zc.recipe.testrunner = 2.0.0
setuptools = 5.4.1

The build is working now, but I'm not sure if it's the proper way? Maybe it's better to remove the external grok.cfg?